external/kafka/src/main/scala/org/apache/gearpump/streaming/kafka/lib/PropertiesConfig.scala

```diff
@@ -16,29 +16,22 @@
  * limitations under the License.
  */
 
-package org.apache.gearpump.streaming.examples.kafka.topn
+package org.apache.gearpump.streaming.kafka.lib
 
-class Rankings[T] extends Serializable {
-  private var rankings: List[(T, Long)] = newRankings
+import java.util.Properties
 
-  def update(r: (T, Long)): Unit = {
-    rankings :+= r
-  }
-  def update(obj: T, count: Long): Unit = {
-    update((obj, count))
-  }
-
-  def getTopN(n: Int): List[(T, Long)] = {
-    rankings.sortBy(_._2).reverse.take(n)
+abstract class PropertiesConfig(properties: Properties) extends java.io.Serializable {
```

Comment:
If what you want is just the function ```putIfAbsent```, you can either put in an util class or inline the code at all.